src/e2e/app.ts

```diff
@@ -790,6 +790,24 @@ export class App {
     return tab.click();
   }
 
+  async searchToolkits(searchText: string): Promise<void> {
+    const document = await this.document();
+    const searchButton = await document.findByRole("button", {
+      name: "Open search",
+    });
+    await searchButton.click();
+    const searchField = await document.findByRole("textbox", {
+      name: "Search",
+    });
+    await searchField.type(searchText);
+    const modalDialog = await document.findByRole("dialog");
+    // const result = await modalDialog.getAllByRole("button");
+    const result = await modalDialog.findAllByRole("heading", {
+      level: 3,
+    });
+    await result[0].click();
```

Comment:
I think it's not clear from the method name that this clicks on the first result.

Maybe split into two?

Reply:
[`cf28c83`](https://github.com/microbit-foundation/python-editor-next/pull/474/commits/cf28c83a6e090bc5b244e6327e4eade9b7356502)